addons/hr_recruitment/i18n/hr_recruitment.pot

```diff
@@ -2509,12 +2509,37 @@ msgstr ""
 
 #. module: hr_recruitment
 #: model:hr.applicant.refuse.reason,name:hr_recruitment.refuse_reason_3
-msgid "The applicant gets a better offer"
+msgid "Refused by Applicant: better offer"
 msgstr ""
 
 #. module: hr_recruitment
 #: model:hr.applicant.refuse.reason,name:hr_recruitment.refuse_reason_2
-msgid "The applicant is not interested anymore"
+msgid "Refused by Applicant: don't like job"
+msgstr ""
+
+#. module: hr_recruitment
+#: model:hr.applicant.refuse.reason,name:hr_recruitment.refuse_reason_4
+msgid "Language"
+msgstr ""
+
+#. module: hr_recruitment
+#: model:hr.applicant.refuse.reason,name:hr_recruitment.refuse_reason_5
+msgid "Role already fulfilled"
+msgstr ""
+
+#. module: hr_recruitment
+#: model:hr.applicant.refuse.reason,name:hr_recruitment.refuse_reason_6
+msgid "Duplicate"
+msgstr ""
+
+#. module: hr_recruitment
+#: model:hr.applicant.refuse.reason,name:hr_recruitment.refuse_reason_7
+msgid "Spam"
+msgstr ""
+
+#. module: hr_recruitment
+#: model:hr.applicant.refuse.reason,name:hr_recruitment.refuse_reason_8
+msgid "Refused by Applicant: salary"
```

Comment:
Could you sort them alphabetically[1]? The easiest would be to export the translation and then do the diff with the current pot file.

[1] : https://github.com/odoo/odoo/wiki/Translations#adding-new-terms-when-suggesting-fixes-in-stable-versions

Reply:
> Could you sort them alphabetically[1]? The easiest would be to export the translation and then do the diff with the current pot file.
> 
> [1] : https://github.com/odoo/odoo/wiki/Translations#adding-new-terms-when-suggesting-fixes-in-stable-versions

Thanks for the review. Now Okay for you ?